### PR TITLE
Created a parameter for ignoring attributes in diffs

### PR DESCRIPTION
**Pull request by Glamhoth**

### CODE REVIEW

File: xmldiff/main.py [hunk 1]
```diff
@@ -113,6 +113,13 @@ def make_diff_parser():
     parser.add_argument(
         "--fast-match", action="store_true", help="A faster, less optimal match run."
     )
+    parser.add_argument(
+        "--ignored-attrs",
+        type=str,
+        nargs="?",
+        help="A comma separated list of attributes "
+        "that should be ignored in comparison. Can be empty.",
```
Comment (regebro) [inline xmldiff/main.py:121]: It's not a required attribute, so we don't need to say that it can be empty, it's implied.

**Comment by regebro** [comment]
Could you add yourself to the contributor list in README.rst as well?
Other than those two small things, this looks good.

**Review by regebro — CHANGES_REQUESTED** [review]

File: xmldiff/main.py (hunk 1 above)
Comment (Glamhoth) [reply xmldiff/main.py]: Removed

**Comment by Glamhoth** [comment]
@regebro conflicts are resolved

**Review by regebro — APPROVED** [review]

**Comment by Glamhoth** [comment]
@regebro conflicts are resolved, MR is ready for merge :]

**Comment by regebro** [comment]
Oh, I forgot that I wanted the parameter to be called `--ignored_attributes`, I'll fix that after the merge.

**Pull request merged**